src/main/java/hotel/models/AvailableDatesList.java

```diff
@@ -0,0 +1,25 @@
+package hotel.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Wrapper class for the list of available dates
+ * @author Ivan Velkushanov
+ */
+public class AvailableDatesList {
+	private List<LocalDate> list = new ArrayList<>();
+
+	public AvailableDatesList(List<LocalDate> list){
+		this.list = list;
+	}
+
+	public List<LocalDate> getList() {
+		return list;
+	}
+
+	public void setList(List<LocalDate> list) {
+		this.list = list;
```

Comment:
might be a good idea to put null check because you initialize the list and then it can became `null` again.